ray-operator/config/webhook/kustomization.yaml

```diff
@@ -4,3 +4,13 @@ resources:
 
 configurations:
 - kustomizeconfig.yaml
+
+patches:
+- patch: |-
+    - op: replace
+      path: /webhooks/0/clientConfig/service/namespace
+      value: ray-system
+  target:
+    kind: ValidatingWebhookConfiguration
+    name: validating-webhook-configuration
+    version: v1
```

Comment:
webhook generator hardcodes the namespace of the service to `system` ([issue](https://github.com/kubernetes-sigs/controller-tools/issues/865)). We change it to `ray-system`